### DISCUSSION TITLE
Separate .qmd -> .html for sections, not just chapters in book?

### DISCUSSION THREAD

**Discussion by graemeblair** [quarto-dev/quarto-cli#1292] [Q&A] [answered]
Hi - thanks for this great project. I'm migrating a book from bookdown to quarto, and was hoping to be able to have some .qmd (and the .html results) be at the chapter level and some be at the section level - a publisher requirement. I have the sidebar set up in this way -- I set it up manually to have contents contain chapters in some parts and chapters with section submenus in others. However, each .qmd with a ## creates a chapter not a section as I had expected. Is there a way to have a .html page in the book be one-per-section for some parts? I think this is similar in spirit to the `split_by` option in bookdown, but would allow it to change part-by-part. Thanks for any recommendations on how to accomplish this!

**Comment by jjallaire** [comment, marked as answer]
There isn't currently a way to split HTML output by anything other than chapter (we are definitely hoping to improve this in the future but it is a limitation for now)

**Reply by graemeblair** [reply]
Thanks very much for the quick reply. Is there some kind of lua filter or other post-processing I could do to accomplish this?

**Comment by jjallaire** [comment]
Not easily. You would need to post-process the raw HTML and fixup all of the links and cross-references as well as the TOCs.

**Reply by graemeblair** [reply]
Thanks!
